server/api/utils/pagination.py

```diff
@@ -54,7 +55,7 @@ async def paginate_permission_filtered_request(
         auth_info: typing.Optional[mlrun.common.schemas.AuthInfo] = None,
         token: typing.Optional[str] = None,
         page: typing.Optional[int] = None,
-        page_size: typing.Optional[int] = None,
+        page_size: typing.Optional[int] = mlconf.httpdb.pagination.default_page_size,
```

Comment:
I understand its ok here since `default_page_size` is not mutable but I think it's better to not use variables as param defaults for the sake of avoiding "gotchas".
```
if page_size is None:
    page_size = mlconf.httpdb.pagination.default_page_size
```